packages/login/src/modules/loginForm/reducer.js

```diff
@@ -0,0 +1,39 @@
+import {SET_MESSAGE, SET_PENDING} from './actions'
+import {LOGIN} from '../actions'
+
```

Comment:
I think we should harmonize the function definitions (don't mix `function setMessage()` with `const setMessage = ...`).